src/http/httpd.cc

```diff
@@ -220,6 +220,10 @@ future<> connection::read_one() {
         }
         ++_server._requests_served;
         std::unique_ptr<http::request> req = _parser.get_parsed_request();
+
+        req->_server_address = _fd.local_address();
```

Comment:
Why `_fd.local_address()`? It should be `this->_server_address`. Same for `_client_address` below